src/content/components/Preferences/Preferences.js

```diff
@@ -27,6 +27,12 @@ export class Preferences extends React.PureComponent {
     prefs.set(e.target.name, e.target.checked);
   }
 
+  // A button for testing iterations
+  async onIterationsCheck() {
+    const iterations = await (await fetch("/api/iterations")).json();
```

Comment:
`await fetch().then(rsp => rsp.json())`

Reply:
😬 not sure where this pattern came from lol